doc/documentation_data_format.rst

```diff
@@ -741,3 +741,118 @@ allows to specify multiple SBML models with corresponding condition and
+should instead be implemented like ``piecewise(par, par > 2, 0)``.
+
+Functions
+~~~~~~~~~
+
+The following functions are supported:
+
+.. csv-table:: Supported functions
+   :header: "Function", "Argument types", "Comment", "Evaluates to"
+
+   "``pow(a, b)``", "float, float", "power function `b`-th power of `a`", "float"
+    "``exp(x)``", "float", "exponential function pow(e, x) (`e` itself not a supported symbol, but ``exp(1)`` can be used instead) ", "float"
+    "``sqrt(x)``", "float", "square root of x; ``pow(x, 0.5)``", "float"
+    "``log(a, b)``, ``log(x)``, ``ln(x)``, ``log2(x)``, ``log10(x)``", "float[, float]", "logarithm of ``a`` with base ``b``; ``log(x, e)``; ``log(x, e)``; ``log(x, 2)``; ``log(x, 10)`` (``log(0)`` is defined as ``-inf``) NOTE: ``log`` without explicit base is ``ln``, not ``log10``", "float"
+    "``sin``, ``cos``, ``tan``, ``cot``, ``sec``, ``csc``", "float", "trigonometric functions", "float"
+    "``arcsin``, ``arccos``, ``arctan``, ``arccot``, ``arcsec``, ``arccsc``", "float", "inverse trigonometric functions", "float"
+    "``sinh``, ``cosh``, ``tanh``, ``coth``, ``sech``, ``csch``", "float", "hyperbolic functions", "float"
+    "``arcsinh``, ``arccosh``, ``arctanh``, ``arccoth``, ``arcsech``, ``arccsch``,", "float", "inverse hyperbolic functions", "float"
+    "``piecewise(true_value_1, condition_1, [true_value_2, condition_2, ] [...] [true_value_n, condition_n,] false_value)``", "``true_value*`` and ``false_value`` are either all float or all bool, ``condition*`` are all bool", "The function value is ``true_value_1`` if ``condition_1`` is true, else ``true_value_2`` if ``condition_2`` is true, else ..., else ``true_value_n`` if ``condition_n`` is true, else ``false_value``.", "float"
+    "``abs(x)``", "float", "absolute value; ``piecewise(x, x>=0, -x)``", "float"
+    "``sign(x)``", "float", "sign of ``x``; ``piecewise(1, x>=0, -1)``", "float"
+    "``min(a, b)``, ``max(a, b)``", "float, float", "minimum / maximum of {a,b}; ``piecewise(a, a<=b, b)``, ``piecewise(a, a>=b, b)``", "float"
+
+Model time
+~~~~~~~~~~
+
+The model time is represented by the symbol ``time``. ``time`` always
+represents the current absolute simulation time (i.e., if the simulation starts
+at :math:`t_0 \neq 0`, then ``time`` is *not* the time since :math:`t_0`).
+
+Identifiers
+-----------
+
+* All identifiers in PEtab must consist only of upper and lower case letters,
+  digits and underscores, and must not start with a digit.
+
+* Identifiers are case-sensitive.
+
+* Identifiers must not be a reserved keyword (see below).
+
+* Identifiers must be globally unique within the PEtab Problem.
+  Function names may be used as identifiers for other model entities, as they
+  can be distinguished by the context.
+
+Reserved keywords
+~~~~~~~~~~~~~~~~~
+
+The following keywords are reserved and must not be used as identifiers:
+
+* ``true``, ``false``: Boolean literals
+* ``inf``: Infinity
+* ``time``: Model time
+* ``nan``: reserved
```

Comment:
```suggestion
* ``true``, ``false``: Boolean literals, used in PEtab expressions.
* ``inf``: Infinity, used in PEtab postequilibration measurements.
* ``-inf``: Negative infinity, used in PEtab timecourses with preequilibration.
* ``time``: Model time, used in PEtab expressions.
* ``nan``: Undefined in PEtab, but reserved to avoid implementation issues.
```

Reply:
Agreed, except for `-inf`. This is unary minus applied to `inf`, rather than a reserved keyword.

Reply:
Hm, IMO we currently treat it as a symbol, not as an expression. e.g. in the current PEtab timecourse proposal, we forbid expressions for the time points, hence the `-inf` is not an expression in a timecourse like `-inf:preeq_cond; 0:sim_cond`, so we have to call it a symbol, right? Or do we say `-inf` is the only allowable expression there? `-inf` also means "simulate from 0 until steady-state", which isn't incompatible with the "expression" interpretation of `-inf`, but feels more natural with the "symbol" interpretation of `-inf`. I guess no problem either way, fine to go with "expression" if we end up wanting to use it in expressions in a future extension.

Reply:
For the timecourses it's fine to treat it as a special symbol. If no expressions are allowed there, this whole section doesn't matter for the start time of the periods anyways. Using `-inf` as identifier for any other entity is anyways not allowed, since it's starting with a minus.